### PR TITLE
feat: Upgrade from issue templates to issue forms

### PR DESCRIPTION
**Pull request by kingthorin**
Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Report a bug impacting ZAP.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a ZAP bug! Fill out the form below to provide us with info to help fix the bug. 
+        Only use this if you're 100% sure you've found a bug and can reproduce it. If you're looking for general help with ZAP, try the [User Group](https://groups.google.com/g/zaproxy-users).
```
Comment (psiinon) [inline .github/ISSUE_TEMPLATE/bug-report.yml:9]: Add something like: "Please do not ask questions as issues - these will be closed." ?

Comment (kingthorin) [reply .github/ISSUE_TEMPLATE/bug-report.yml:9]: Sure

**Comment by kingthorin** [comment]
Tweaked.

File: .github/ISSUE_TEMPLATE/fature-request.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+name: Feature Request
```
Comment (psiinon) [inline .github/ISSUE_TEMPLATE/fature-request.yml:1]: File name typo - should be "feature" ;)

Comment (kingthorin) [reply .github/ISSUE_TEMPLATE/fature-request.yml:1]: Doh

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Poke @ricekot or @thc202 😉

**Comment by thc202** [comment]
Guess the old links will stop working?

**Comment by kingthorin** [comment]
We had linked to specific templates?
I guess so :shrug:

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        If applicable, add screenshots to help explain your enhancement/idea.
+
+        Tip: You can attach files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here.
```
Comment (thc202) [inline .github/ISSUE_TEMPLATE/feature-request.yml:43]: problem → enhancement ?

**Comment by kingthorin** [comment]
I could restore the old templates and put in placeholder content that points to the new ones?

**Comment by thc202** [comment]
We had and they were posted in the mailing list (and might be in docs?). I'm not too worried just checking.

**Comment by kingthorin** [comment]
Okay. It seems like users are presented with the blank form if the template isn't found. So on the rare occasion we'll just have to deal with it or ask the user to redo it.

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**